### DISCUSSION TITLE
ARDUINO RP2040  CONNECT + WAVESHARE ILI9486 DISPLAY

### DISCUSSION THREAD

**Discussion by aroesz98** [Bodmer/TFT_eSPI#1289] [Compatible displays and setup files]
Hello.
Is there and way to run this configuration? I'm using arduino-pico earle philhower's core but display doesn't seems to work.
I configured User_Setup file to my pinout but still nothing. For any help i will be very greatfull.
Regards :).

**Comment by spinalcode** [comment]
Did you ever figure this out? I have an ili9486 spi screen meant for a raspberry pi that I would like to use with my Pico but can't figure it out 😟

**Reply by Bodmer** [reply]
It depends on the exact model of display, there are a few variants. See #1691 which is now supported.

**Comment by spinalcode** [comment]
Got the screen 'working' but there seems to be something up with the pixel format... https://socoder.net/uploads/124/2022/13/PXL_20220331_064537441.jpg
